ws-client/src/manager.rs

```diff
@@ -1,9 +1,11 @@
 //! Handles and monitors JSONRPC v2 method calls and subscriptions
 //!
 //! Definitions:
-//!     * Request ID - request ID in JSONRPC-v2 specification
-//!     (the specs allow number, string or null but this crate only supports numbers)
-//!     * Subscription ID - unique ID generated by server
+//!
+//! 	- RequestId: request ID in JSONRPC-v2 specification
+//! 		> **Note**: The spec allow number, string or null but this crate only supports numbers.
```

Comment:
What's up with the `>` and funny indentation here?

Reply:
It looked very bad in the generated docs....

Reply:
my bad I used tabs in `doc comments`